Robinhood/Robinhood.py

```diff
@@ -367,6 +381,7 @@ def print_quote(self, stock=''):    # pragma: no cover
             Returns:
                 None
         """
+        print(self.session.get(endpoints.orders(),timeout=15).json)
```

Comment:
Without digging much into this (it's been a while) I imagine that get request is returning a response that you are trying to parse using `.json` - I believe that is a method on the `requests` module and therefore needs to be invoked right? Like `.json()`?